core/controllers/story_editor.py

```diff
@@ -161,7 +175,14 @@ def get(self, story_id: str) -> None:
 
     @acl_decorators.can_edit_story
     def put(self, story_id: str) -> None:
-        """Updates properties of the given story."""
+        """Updates properties of the given story.
+
+        Args:
+            story_id: str. The story ID.
+
+        Raises:
+            InvalidInputException. Error class for invalid input.
```

Comment:
See above on how to document this properly.

Reply:
I have made the changes.